src/azure-cli/azure/cli/command_modules/cosmosdb/_help.py

```diff
@@ -1655,3 +1655,66 @@
     text: |
       az cosmosdb service delete --resource-group MyResourceGroup --account-name MyAccount --name "sqlDedicatedGateway"
 """
+
+helps['cosmosdb sql database restore'] = """
+type: command
+short-summary: Restore a deleted sql database within the same account.
+examples:
+  - name: Restore a deleted sql database within the same account.
+    text: |
+      az cosmosdb sql database restore --resource-group resource_group --account-name database_account_name --name name_of_database_needs_to_be_restored --restore-timestamp 2020-07-13T16:03:41+0000
+"""
+
+helps['cosmosdb sql container restore'] = """
+type: command
+short-summary: "Restore a deleted sql container within the same account."
```

Comment:
You actually don't need `""` for the summary string

Reply:
updated as per suggestion